src/Avalonia.Themes.Default/Accents/BaseLight.xaml

```diff
@@ -2,29 +2,59 @@
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:sys="clr-namespace:System;assembly=mscorlib">
   <Style.Resources>
-    <SolidColorBrush x:Key="ThemeBackgroundBrush">#FFFFFFFF</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeBorderLightBrush">#FFAAAAAA</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeBorderMidBrush">#FF888888</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeBorderDarkBrush">#FF333333</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeControlLightBrush">#FFFFFFFF</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeControlMidBrush">#FFAAAAAA</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeControlDarkBrush">#FF888888</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeForegroundBrush">#FF000000</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeForegroundLightBrush">#FF808080</SolidColorBrush>
+      
+    <Color x:Key="ThemeAccentColor">#CC119EDA</Color>
+    <Color x:Key="ThemeAccentColor2">#99119EDA</Color>
+    <Color x:Key="ThemeAccentColor3">#66119EDA</Color>
+    <Color x:Key="ThemeAccentColor4">#33119EDA</Color>
+      
+    <Color x:Key="ThemeBackgroundColor">#FFFFFFFF</Color>
+    <Color x:Key="ThemeBorderLightColor">#FFAAAAAA</Color>
+    <Color x:Key="ThemeBorderMidColor">#FF888888</Color>
+    <Color x:Key="ThemeBorderDarkColor">#FF333333</Color>
+    <Color x:Key="ThemeControlLightColor">#FFFFFFFF</Color>
+    <Color x:Key="ThemeControlMidColor">#FFAAAAAA</Color>
+    <Color x:Key="ThemeControlDarkColor">#FF888888</Color>
+    <Color x:Key="ThemeControlHighlightLowColor">#FFF0F0F0</Color>
+    <Color x:Key="ThemeControlHighlightMidColor">#FFD0D0D0</Color>
+    <Color x:Key="ThemeControlHighlightDarkColor">#FF808080</Color>
+    <Color x:Key="ThemeForegroundColor">#FF000000</Color>
+    <Color x:Key="ThemeForegroundLightColor">#FF808080</Color>
+      
+    <Color x:Key="HighlightColor">#FF086F9E</Color>
+    <Color x:Key="ErrorColor">#FFFF0000</Color>
+    <Color x:Key="ErrorLightColor">#10FF0000</Color>
 
-    <SolidColorBrush x:Key="HighlightBrush">#FF086F9E</SolidColorBrush>    
-    <SolidColorBrush x:Key="ThemeAccentBrush">#CC119EDA</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeAccentBrush2">#99119EDA</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeAccentBrush3">#66119EDA</SolidColorBrush>
-    <SolidColorBrush x:Key="ThemeAccentBrush4">#33119EDA</SolidColorBrush>
-    <SolidColorBrush x:Key="ErrorBrush">Red</SolidColorBrush>
-    <SolidColorBrush x:Key="ErrorBrushLight">#10ff0000</SolidColorBrush>
+    <Color x:Key="InvisibleSelectionColor">#01000000</Color>
```

Comment:
What is `InvisibleSelectionColor` needed for and why is its color `#01000000`? That seems like a strange value. Why not use `Transparent`?

Reply:
It's actually quite strange and something I picked up working a lot with WPF in the past... Transparent on (some) controls (see Border) don't track the cursor/pointer events, so the truck is to make the background near invisible to get around this.

I'll post a demo/gif later.

Reply:
Hmm, I don't see why that would be the case. If you can find a simple repro, please let me know!

Reply:
Lol, turns out this work around is no longer needed, it did however give me trouble earlier.